tests/acceptance/features/bootstrap/DataExporterContext.php

```diff
@@ -256,14 +280,47 @@ private function assertFilePhysicallyExistInLastExport($filename, $message = '')
 	/**
 	 * @param string $filename
 	 * @param string $content
+	 * @param bool $trash
 	 *
 	 * @return void
 	 */
-	private function assertFilePhysicallyExistInLastExportWithContent($filename, $content) {
-		$this->featureContext->theFileWithContentShouldExistInTheServerRoot(
-			self::path("$this->lastExportPath/files/$filename"),
-			$content
-		);
+	private function assertFilePhysicallyExistInLastExportWithContent($filename, $content, $trash = false) {
+		if ($trash) {
+			$this->featureContext->listTrashbinFileInServerRoot(self::path("$this->lastExportPath/files_trashbin"));
+			$matches = [];
+			foreach (HttpRequestHelper::getResponseXml($this->featureContext->getResponse())->data->element as $item) {
+				$file = (string)$item;
+				if (\strpos($file, $filename) === 0) {
+					$matches[] = $file;
+				}
+			}
+			$fileFound = false;
+			foreach ($matches as $match) {
+				$path = "$this->lastExportPath/files_trashbin/$match";
+				$this->featureContext->readFileInServerRoot($path);
+				Assert::assertSame(
+					200,
+					$this->featureContext->getResponse()->getStatusCode(),
+					"Failed to read the file {$path}"
+				);
+				$fileContent = HttpRequestHelper::getResponseXml($this->featureContext->getResponse());
+				$fileContent = (string)$fileContent->data->element->contentUrlEncoded;
+				$fileContent = \urldecode($fileContent);
+
+				if ($fileContent === $content) {
+					$fileFound = true;
```

Comment:
maybe, instead of naming the variable `$fileContent` you should simply call it `$content` and access it with `$this->$content` !! is this what you wanted to say @phil-davis ?

Reply:
maybe, instead of naming the variable `$fileContent` you should simply name it `$content` and access that with `$this->$content` !!
Is this what you wanted to say @phil-davis ?

Reply:
```suggestion
					$fileFoundWithMatchingContent = true;
```
Maybe a name like this?